### DISCUSSION TITLE
Stripe Checkout

### DISCUSSION THREAD

**Discussion by codewebs-org** [lunarphp/lunar#1133] [Feature Requests]
Will Stripe Checkout be supported ?

**Comment by glennjacobs** [comment]
Being a headless system, we naturally don't offer a frontend integration. The Stripe addon we have may be compatible with a Stripe Checkout implementation <https://github.com/lunarphp/stripe>, or you may be able to tweak it to your needs.

**Comment by yasirmturk** [comment]
How to make stripe addon licensed? 
`This addon has not been licensed, please check the config for this addon.`
<img width="601" alt="Screenshot 2023-08-27 at 5 20 08 PM" src="https://github.com/lunarphp/lunar/assets/306691/8479be2e-927c-427c-b3d2-36b3c06b9f54">

**Reply by glennjacobs** [reply]
Licensing was an idea we had, but never actually happened. With the Filament hub coming later this year, I think we may look to remove this. I'll speak with @alecritson and come back to you.

**Reply by yasirmturk** [reply]
how can I disable it until then?

**Reply by glennjacobs** [reply]
There is no option currently.

**Reply by yasirmturk** [reply]
is there any other working plugin for payments via stripe/paypal?

**Reply by glennjacobs** [reply]
In v0.6 which will be released in the next week or so, the licensing has been removed.

**Reply by yasirmturk** [reply]
Super thanks
